generators/aws-containers/README.md

```diff
@@ -0,0 +1,24 @@
+# AWS JHipster Generator - _A new hope_
+
+## AWS
+### ECR
+#### Authentication Token
+The token received by ECR to authenticate is structured as follow. We'll only take the first object.
```

Comment:
`follow` -> `followed`

Reply:
It's actually 'as follows' but thanks for the catch up :)

Reply:
Right!! thanks